config/module.config.php

```diff
@@ -31,6 +31,12 @@
                 'hydrator_dir'       => 'data/DoctrineMongoODMModule/Hydrator',
                 'hydrator_namespace' => 'DoctrineMongoODMModule\Hydrator',
 
+                'generate_persistent_collections' => Configuration::AUTOGENERATE_ALWAYS,
```

Comment:
I mean other Doctrine-related Zend modules. And I must disagree that app is broken by default, in Symfony's case when cache is not there, it's generated and so are proxies/hydrators because that's hooked into cache-warming process by the bundle so the app is never broken by default.